helm-chart/templates/cronjob.yaml

```diff
@@ -0,0 +1,111 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  labels:
+    app: popeye-reports
+  name: popeye-reports
+  namespace: {{ .Values.namespace }}
+spec:
+  schedule: {{ .Values.schedule }}
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: {{ .Values.failedJobsHistoryLimit | default "20" }}
+  successfulJobsHistoryLimit: {{ .Values.successfulJobsHistoryLimit | default "20" }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: popeye-reports
+        spec:
+          containers:
+          - image: "{{ .Values.image.web.name }}:{{ .Values.image.web.tag }}"
+            imagePullPolicy: {{ .Values.image.web.pullPolicy }}
+            name: nginx
+            ports:
+            - containerPort: 80
+              name: http
+              protocol: TCP
+            resources: {}
+            terminationMessagePath: /dev/termination-log
+            terminationMessagePolicy: File
+            volumeMounts:
+            - mountPath: /usr/share/nginx/html
+              name: www
+          - image: "{{ .Values.image.recycle.name }}:{{ .Values.image.recycle.tag }}"
```

Comment:
This container should cleanup the previously exported reports in the nginx-volume.